packages/graphql/src/translate/queryAST/ast/operations/ReadOperation.ts

```diff
@@ -100,47 +100,127 @@ export class ReadOperation extends Operation {
         return filterTruthy(this.authFilters.map((f) => f.getPredicate(context)));
     }
 
+    private transpileWithCreateSelection(context: QueryASTContext): OperationTranspileResult {
```

Comment:
If I understand correctly, here you have moved all the transpile for create operations flow to a separate method.

I'm not sure if that is the best approach for future refactors. As the idea would be to make the code the same between create and non-create, and having this as a separate flow leads to more 

Thoughts? Or maybe I'm missing something

Reply:
This change has been removed from this PR now, and will be addressed in a future refactor